### DISCUSSION TITLE
security issue

### DISCUSSION THREAD

**Discussion by catty-dev** [alfiankan/teleterm#21] [Ideas]
Literally everybody can access the server through the bot and all bots are public with an @. In terms of functinality the bot is great but you need to add a Whitlist for users

**Comment by alfiankan** [comment]
Thanks for reporting the security issue.
  It’s very good idea implementing a whitelist, I'll be adding that feature next weekend, thanks again

**Comment by catty-dev** [comment]
Nice to hear. I'm lookin forward to it. Merry christmas!

**Comment by alfiankan** [comment]
new release https://github.com/alfiankan/teleterm/releases/tag/v2.1.0



## What's Changed
* alfajrimutawadhi/feature/telegram-userid-whitelist by @alfajrimutawadhi in https://github.com/alfiankan/teleterm/pull/18

## New Contributors
* @alfajrimutawadhi made their first contribution in https://github.com/alfiankan/teleterm/pull/18

**Full Changelog**: https://github.com/alfiankan/teleterm/compare/v2.0.0...v2.1.0


## Note
- v2.1.0 using userid for whitelisting access to the bot
- username base whitelisting will be new feature in v2.2.0 https://github.com/alfiankan/teleterm/issues/17
